src/activities/assignments/AssignmentEntity.js

```diff
@@ -194,11 +194,8 @@ export class AssignmentEntity extends Entity {
 		if (!this._entity) {
 			return false;
 		}
-		const subEntity = this._entity.getSubEntityByRel(Rels.Assignments.folderType);
-		if (!subEntity) {
-			return false;
-		}
-		return subEntity.hasClass(Classes.assignments.assignmentType.readOnly);
+		return !this._entity.hasActionByName(Actions.assignments.setToGroup) &&
```

Comment:
These actions got moved to the main assignment entity?

Reply:
Nope that is my bad, pushing a fix